examples/lenet/index.html

```diff
@@ -0,0 +1,122 @@
+			margin-right:8px;
+      position:relative;
+			vertical-align: middle;
+
+		}
+		.radioInit:checked + .radioSelected:after{
+			content:"";
+			display: inline-block;
+			width:22px;
+			height:22px;
+			background:lightskyblue;
+			border-radius:100%;
+			margin-top:2px;
+		}
+		.radioInit:checked + .radioSelected{
+			border:2px solid rgb(21, 133, 202);
+		}
+	  </style>
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-md navbar-dark bg-dark" style="width:100%">
+      <span class="navbar-brand mb-0 h1">WebNN LeNet Demo</span>
+    </nav>
+    <div class="container" id="container">
+      <div class="row" style="margin-top:20px;">
+        <div class="col">
+        </div>
+        <div class="col-4" align="center">
+          <canvas id="visual_canvas" width=280 height=280></canvas>
+        </div>
+        <div class="col-4" align="center">
+          <div class='row text-center tc' id='inferencetitle'>
+            <h5>Prediction Result:</h5>
+          </div>
+          <div class='row text-center' id='inferenceresult'>
+            <table class='table table-sm'>
+              <thead>
+                <tr>
+                  <th scope='col'>#</th>
+                  <th scope='col'>Label</th>
+                  <th scope='col'>Probability</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <th scope='row'>1</th>
+                  <td id='label0'></td>
+                  <td id='prob0'></td>
+                </tr>
+                <tr>
+                  <th scope='row'>2</th>
+                  <td id='label1'></td>
+                  <td id='prob1'></td>
+                </tr>
+                <tr>
+                  <th scope='row'>3</th>
+                  <td id='label2'></td>
+                  <td id='prob2'></td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+        <div class="col">
+        </div>
+      </div>
+      <div class="row" style="margin-top:50px;">
+        <div class="col">
+        </div>
+        <div class="col-1">
+          <button type="button" class="btn btn-primary btn-lg" id="next">Next</button>
+        </div>
+        <div class="col-1">
+          <button type="button" class="btn btn-primary btn-lg" id="clear">Clear</button>
+        </div>
+        <div class="col-1">
+          <button type="button" class="btn btn-primary btn-lg" id="predict" disabled>Predict</button>
+        </div>
+        <div class="col-4"></div>
+        <div class="col">
+        </div>
+      </div>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/js/bootstrap.min.js"></script>
+    <script src="../../dist/webml-polyfill.js"></script>
+    <script src="lib/mnist.js"></script>
```

Comment:
@huningxin Now **mnist.js** is under mnist folder, you maybe forgot to update this src path after you renamed lib to mnist, please update 
```js
src="lib/mnist.js"
```
to
```js
src="mnist/mnist.js"
```

Reply:
good catch. I'll fix it. Thanks!

Reply:
Fixed in the latest commits. Please take another look.